.NET/Microsoft.Recognizers.Definitions/English/DateTimeDefinitions.cs

```diff
@@ -160,8 +160,8 @@ public static class DateTimeDefinitions
 		public const string PeriodTimeOfDayRegex = @"\b((in\s+(the)?\s+)?((?<early>early(\s+|-))|(?<late>late(\s+|-)))?(?<timeOfDay>morning|afternoon|night|evening))\b";
 		public static readonly string PeriodSpecificTimeOfDayRegex = $@"\b(({StrictRelativeRegex}\s+{PeriodTimeOfDayRegex})\b|\btonight)\b";
 		public static readonly string PeriodTimeOfDayWithDateRegex = $@"\b(({TimeOfDayRegex}(\s+(on|of))?))\b";
-		public const string LessThanRegex = @"\b(less\s+than)\s+$";
-		public const string MoreThanRegex = @"\b(more\s+than)\s+$";
+		public const string LessThanRegex = @"\b(less\s+than)\b";

```

Comment:
These two Regexes now not only apply to cases like "3 days or more than"
But also "more than 3 days"